packages/core/mixins/_underline.scss

```diff
@@ -0,0 +1,11 @@
+@mixin underline-color($color, $focus-color) {
```

Comment:
Tror kanskje jeg ville lagt denne i `_ornaments.scss` e.l. i stedet for å lage egen fil, men det er ikke krise 😊